### DISCUSSION TITLE
Request for adding to HacktoberFest Topic

### DISCUSSION THREAD

**Discussion by nkitsaini** [svsool/vscode-memo#603] [General]
[Hacktoberfest](https://hacktoberfest.com/) is an annual event where individuals try to make open source contributions in month of October. Starting this year hacktoberfest will only count contributions to repositories in [HacktoberFest Topic](https://github.com/topics/hacktoberfest).

I would like to contribute to this project (specially making it work on vscode web :crossed_fingers: ). And I would really appreciate if you can add this project to the HacktoberFest Topic. This might also attract contribution from other participants.

**Comment by svsool** [comment]
Hi @nkitsaini, it should be included now.
